### PR TITLE
iconGrid: sync editable and selectable properties of ClutterText

### PR DESCRIPTION
**Pull request by mcatanzaro**
With clutter 1.26, ClutterText will never propagate button press events
if it is selectable. We don't actually need our ClutterText to be
selectable except when we are in edit mode, so keep the editable and
selectable properties in sync. This fixes a regression caused by clutter
1.26 where it became impossible to rename desktop icons.

https://phabricator.endlessm.com/T12228

### CODE REVIEW

File: js/ui/iconGrid.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ const EditableLabel = new Lang.Class({
         this.parent(params);
 
         this.clutter_text.editable = false;
+        this.clutter_text.selectable = false;
```
Comment (ebassi) [inline js/ui/iconGrid.js]: Drive-by comment — you could even do:

```
this.clutter_text.bind_property('editable', this.clutter_text, 'selectable', GObject.BindingFlags.BIDIRECTIONAL | GObject.BindingFlags.SYNC_CREATE);
```

inside the instance initialization, so you would not need the rest of the patch.

Comment (mcatanzaro) [inline js/ui/iconGrid.js]: OK, that seems safer, so we don't ever get these desynced in the future.
